apps/dashboard/features/holders-and-delegates/components/TokenHolders.tsx

```diff
@@ -420,12 +420,15 @@ export const TokenHolders = ({
           />
         </div>
       </div>
-        <HoldersAndDelegatesDrawer
-          isOpen={!!selectedTokenHolder}
-          onClose={handleCloseDrawer}
-          entityType="tokenHolder"
-          address={selectedTokenHolder || "0x0000000000000000000000000000000000000000"}
-        />
+      <HoldersAndDelegatesDrawer
+        isOpen={!!selectedTokenHolder}
+        onClose={handleCloseDrawer}
+        entityType="tokenHolder"
+        address={
+          selectedTokenHolder || "0x0000000000000000000000000000000000000000"
```

Comment:
when is the `selectedTokenHolder` null? 
and if there is actually a scenario, use `zeroAddress` variable from Viem